app/controllers/agent_statuses_controller.rb

```diff
@@ -2,7 +2,14 @@
 class AgentStatusesController < ApplicationController
   def index
     @agent_statuses = AgentStatus.where(open: true).joins(agent: :team)
-    @agent_statuses.each { |a| a.status = normalize_status(a.status) }
+    lunched = Rails.cache.read 'lunched'
+    lunched ||= Set.new
+    @agent_statuses.each do |a|
+      a.status = normalize_status(a.status)
+      a.lunch = false
+      a.lunch = true if lunched.include? a.agent_id
```

Comment:
It would be a bit cleaner to use a single statement here, after all `lunched.include? a.agent_id` returns a boolean.